console.log("let's learn try-catch");
console.log(`hello mohib`);

const funct1 = (p) => {
  console.log(`I am`,p);
};

console.log(funct1("aman"));

const alertf = (params) => {
    alert("hey")
}
try {
    console.log("hey")
    console.log(alertff());
    console.log("done")
} catch (error) {
  console.log(error);
}

console.log("bye mohib")
